app/persistence/postgreSQL/PgService.ts

```diff
@@ -97,10 +97,26 @@ export class PgService {
 	 * @memberof PgService
 	 */
 	getUserModel(attributes, options) {
-		const sequelize = new Sequelize(
-			`postgres://${this.pgconfig.user}:${this.pgconfig.password}@${this.pgconfig.host}:${this.pgconfig.port}/${this.pgconfig.database}`,
-			{ logging: false }
-		);
+		
+		// Add SSL option for PostGresql- Issue #404
+		const isPostgresSslEnabled = process.env.DATABASE_SSL_ENABLED || false;
+
+		if (isPostgresSslEnabled) {
+			logger.info('SSL to Postgresql enabled: set dialect SSL option');
+			const sequelize = new Sequelize(
+				`postgres://${this.pgconfig.user}:${this.pgconfig.password}@${this.pgconfig.host}:${this.pgconfig.port}/${this.pgconfig.database}`,
+				{ logging: false, dialectOptions: { ssl: true, }, }
+			);
+		}else{
+			logger.info('SSL to Postgresql disabled: dialect options not set');
+			const sequelize = new Sequelize(
+				`postgres://${this.pgconfig.user}:${this.pgconfig.password}@${this.pgconfig.host}:${this.pgconfig.port}/${this.pgconfig.database}`,
+				{ logging: false }
+			);
+
+		}
+
+
 		this.userModel = sequelize.define('users', attributes, options);
```

Comment:
Please change the scope of 'sequelize' so that it can be accessed in the line 120.

Reply:
updated code - kindly verify

Reply:
@zicaden Thanks for the update.  Need to discuss about the SSL enable functionality. Can you please join the maintainers by-weekly call? If so, it would be greatly appreciated!

Reply:
Hello - Please share the details of the call

Reply:
@Udhayakumari - tagging you. Please let me know the details of the call. Also I found some new issues - the SSL configuration is better to be done using environment variables - 

e.g. DATABASE_MTLS_ENABLED = true | false
If false - then we can avoid placing the certificate and key (key: | cert:) 

We encountered issue while connecting with Azure Postgres Flexible Server where these parameters are causing issues. If you want I can provide you an updated code with these changes.

Reply:
@zicaden Thanks for the update! To join the meeting please find the "Blockchain Explorer - Contributors Meeting" in this Calendar link: [https://lists.hyperledger.org/g/labs/calendar](url)

Reply:
@Udhayakumari - Hello I joined the meeting today but no one joined. 

    Wednesday, April 17th, 2024
    2:00pm to 3:00pm
    (UTC) Coordinated Universal Time 
Repeats:
    Every 2 weeks on Wednesday 
Location:
    https://zoom.us/my/hyperledger.community?pwd=STZQd0xMZU9xRVVOVnpQM3JNQ2dqZz09 
Organizer:
    Blockchain Explorer Maintainers 
Description:
    Blockchain Explorer - Hyperledger Labs project weekly contributors meeting.

    Meeting ID: 403 498 3298
    Passcode: 475869
    https://zoom.us/j/4034983298?pwd=STZQd0xMZU9xRVVOVnpQM3JNQ2dqZz09

Reply:
@zicaden Yesterday call has been cancelled. Sorry for the inconvenience!

Reply:
@Udhayakumari - please confirm if this can be discussed in your next meeting scheduled on 4th Sep, 2024 at 7:30 PM IST? If not, kindly confirm the specific meeting date / time when you can be available.